### ISSUE TITLE
The role assignment is confusing in  11_vs_11_{}_stochastic SCENARIO

### ISSUE THREAD

**Issue by Atan03** [google-research/football#320]
The **initial positions** of the players in 11_vs_11_{}_stochastic are shown as below：
![捕获](https://user-images.githubusercontent.com/46615197/173177748-8fa2cad0-65af-4473-b92c-ba061a3b5557.PNG)
As you can see，the player **3，4，5，6**  are in the backline.
However,during the game, the re-initialising positions are not corresponding these initial positions:
![捕获](https://user-images.githubusercontent.com/46615197/173178132-7ecfe546-6ecf-4cc8-956e-70479296f6b7.PNG)
As you can see，the player **1，2，3，4**  are in the backline this time.
A more visual example, the scenario 11_vs_11 stochastic assign player 9 **the role of a CM**, but puts it in **the CF position** when re-initialising as the screenshot of step 1555 shown.
![捕获](https://user-images.githubusercontent.com/46615197/173178297-10d4c292-e1d0-4537-8011-f7c2dccfe3ec.PNG)
I carefully checked all the 11_vs_11 scenarios under the folder, and I found that **only the 11_vs_11 kaggle assigns the right roles**  corresponding to the positions after re-initialising.
![捕获](https://user-images.githubusercontent.com/46615197/173178448-fc7060f6-15e3-40c5-b5ec-b04db984e201.PNG)
As you can see ，the player 1，2，3，4 are in the backline and the player 9 is assigned the role of CF.

I compare the build-in AI’s behaviour under 11_vs_11_kaggle and 11_vs_11_stochastic，the AIs in former scenario seem more reasonable and keep a better formations. So i think that the role of player may be important to the rule-based build-in AI.

For the training phase, this is also unreasonable if we add the role information in the feature.

To sum up, I suggest that revising the scenario file of 11_vs_11_?_stochastic to the same as 11_vs_11 kaggle.

**Comment by kaustesseract** [comment]
Hi @Atan03 
Could you please tell me how did you set up the left and right teams as built in AI? I have to make two AI-bots play so I am trying to set both left and right teams as AI bots. When I am trying to run the simulation by passing action the default action as 19 (as instructed in the observation.md file) it is giving me error. It would be very helpful if you can share the configuration details.

n_episodes=5
for e in tqdm(range(n_episodes)):
    ret, obs, done = 0, env.reset(), False
    while not done:
        #actions = env.action_space.sample()
        obs, rew, done, info = env.step(19)
    print(f"Episode={e}, Return={ret}")

**Comment by kaustesseract** [comment]
Hi @Atan03. Thank you so much for this. It will really help me a lot. I am attaching below 2 screenshots:
 1. Custom made environment 
 2. The create_environment function
It will be really helpful if you can look at it and let me know if I have done the setup correctly. 

![Screenshot from 2022-08-05 22-36-16](https://user-images.githubusercontent.com/17180036/183078468-ee14ac10-77fb-46cd-b7ed-d31dc6d0b38d.png)

![Screenshot from 2022-08-05 22-37-44](https://user-images.githubusercontent.com/17180036/183078809-70c2e3f6-7654-4913-a9d2-1b44606fe40d.png)
